### DISCUSSION TITLE
Interpret hd.tooltip content as markdown

### DISCUSSION THREAD

**Discussion by tkriplean** [hyperdiv/hyperdiv#41] [Ideas]
Just an idea. Often when I'm using an info icon + tooltip to explain the interface or context more, it would help to provide some baseline formatting to structure the information. Interpreting the tooltip string as markdown rather than pure text would help facilitate that structuring, without, I think, interfering much with other pure text tooltips.

**Comment by mariusnita** [comment]
I'll add support for a `content_slot` in tooltip. You'd be able to do:

```py
with hd.tooltip() as tooltip:
    hd.markdown("# Markdown", slot=tooltip.content_slot)
    hd.button("Hover Me")
```

You can put boxes or other components in that slot, too.
